src/transaction/log_record.hpp

```diff
@@ -244,7 +249,10 @@ struct log_rec_ha_server_state
   int state;			/* ha_Server_state */
   int dummy;			/* dummy for alignment */
 
-  INT64 at_time;		/* time recorded by active server */
+  time_msec_t at_time;		/* time recorded by active server
+                                 * NOTE: stored as milliseconds with the value as accurate
+                                 * as possible as provided via system structs such as
+                                 * 'timeval' or 'timespec' */
```

Comment:
I think the comment explanation is not useful for the future reader. The type `time_msec_t` is self-explanatory.
```suggestion
  time_msec_t at_time;		/* time recorded by active server */
```

Reply:
Removed